deploy/examples/create-external-cluster-resources.py

```diff
@@ -768,7 +869,7 @@ def get_ceph_dashboard_link(self):
 
     def create_rgw_admin_ops_user(self):
         cmd = ['radosgw-admin', 'user', 'create', '--uid', self.EXTERNAL_RGW_ADMIN_OPS_USER_NAME, '--display-name',
-               'Rook RGW Admin Ops user', '--caps', 'buckets=*;users=*;usage=read;metadata=read;zone=read']
+               'Rook RGW Admin Ops user', '--caps', 'info=read;buckets=*;users=*;usage=read;metadata=read;zone=read']
```

Comment:
I think we should only add this if the cluster version is at least Quincy otherwise the script will always fail.
We need to:

* Check the local binary version with `ceph -v`
* Check the cluster's version with `ceph version`

If not Qunicy, don't add the `info=read` and log a message that we won't be able to validate the rgw endpoint.

Reply:
The `info=read` PR is been backporting to pacific, I don't think so we would need to verify it then, at least for 4.10  will work fine then.
https://tracker.ceph.com/issues/51432